bucket/winssh-pageant.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.2",
+    "description": "Pageant to Windows OpenSSH agent bridge",
+    "homepage": "https://github.com/ndbeals/winssh-pageant",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_amd64.zip",
+            "hash": "a9bef9d134e872e60fd1d0950ce29f73b98af4802c9e93987a4ecafe8debf814"
+        },
+        "32bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_386.zip",
+            "hash": "0591f0c4475dba8fbfacc7ffe5fcb77ac384008d884976b386b4f25fa894e3a8"
+        }
+    },
+    "bin": "winssh-pageant.exe",
+    "notes": [
```

Comment:
Notes are essential information for users. Does not make sense to hide them in manifest.

Put it above architecture property

Reply:
I'm doing the exact same thing https://github.com/lukesampson/scoop-extras/blob/master/bucket/ssh-agent-wsl.json is doing, echoing a command that users can run (or add to an autoran script, same thing). Why is that utility allowed to do it?